### PR TITLE
Show count beside each sound filter

### PR DESCRIPTION
**Pull request by woofers**
Shows the number of samples each sound filter will have prior to selecting.  Also hides filters with no sounds.

change

### CODE REVIEW

File: renderer/components/samples.tsx [hunk 1]
```diff
@@ -62,9 +90,45 @@ const Samples = ({ sounds = [] }: { sounds: Sound[] }) => {
...
+        // Not a great implementation here since its essentially
+        // synchronous but the IPC event emitter returns the first response
+        // and returns the same result for all calls.
+        for (const [feature, value] of pairs) {
+            const count = await getCount(feature, value);
+            if (!counts[feature]) counts[feature] = {};
+            counts[feature][value] = count;
+        }
+        return counts;
+    };
```
Comment (julesyoungberg) [inline renderer/components/samples.tsx:115]: I think we would need to generate a unique response channel for each query in order to do it asynchronously.

Comment (julesyoungberg) [reply renderer/components/samples.tsx:115]: Another option could be to send a list of queries and stream the results back as is done for the analyzer.

**Review by julesyoungberg — APPROVED** [review]
Nice :+1:

File: renderer/components/samples.tsx (hunk 1 above)
Comment (woofers) [reply renderer/components/samples.tsx:115]: Yea it seems to perform fine with a large amount of sounds so I left it for now but we could proabbly do better here

**Pull request merged**